docs/topics/class-based-views/intro.txt

```diff
@@ -173,29 +173,6 @@ that inherits from ``View`` - for example, trying to use a form at the top of a
 list and combining :class:`~django.views.generic.edit.ProcessFormView` and
 :class:`~django.views.generic.list.ListView` - won't work as expected.
 
-.. _mixins_that_wrap_as_view:
-
```

Comment:
I guess this technique could still be useful for other things. We can just point out the existence of the new LoginRequiredMixin which is more customizable.